file.c

```diff
@@ -423,7 +423,7 @@ warnpos(File *f, int adj, char *fmt, ...)
     va_list ap;
 
     off = lseek(f->fd, 0, SEEK_CUR);
-    fprintf(stderr, "%s:%u: ", f->path, off+adj);
+    fprintf(stderr, "%s:%d: ", f->path, off+adj);
```

Comment:
@ysmolsky do you know why the data type changed here?

Reply:
The type of "off+adj" was signed int, so we just use the correct formatting directive to avoid possible confusion.